include/grpc/grpc_crl_provider.h

```diff
@@ -63,10 +63,21 @@ class CrlProvider {
   // Get the CRL associated with a certificate. Read-only.
   virtual std::shared_ptr<Crl> GetCrl(
       const CertificateInfo& certificate_info) = 0;
+  // Returns whether a revocation status of `Undetermined` should be treated as
+  // `Revoked` (deny connections) or `Unrevoked` (allowing connections).
+  // Not finding a CRL is a specific behavior. Per RFC5280, not having a CRL to
```

Comment:
When reading this paragraph, I found "Not finding a CRL is a specific behavior." confusing until I read the rest. Should we consider removing this sentence, since the same idea is repeated in more detail in the next sentence?

Reply:
I agree, done